### PR TITLE
2671 update virtual storage video placements

### PR DESCRIPTION
**Pull request by A-Wheeto**
## Status

* Current Status: Ready for (front-end / tech) review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2671

## Review progress:

- [ ] Browser tested
- [ ] Front-end review completed
- [ ] Tech review completed

## What's changed?

- Added new option for button on banner component
- Rspec test for component
- Updated pages

### CODE REVIEW

File: app/views/pages/careers-support.html.erb [hunk 1]
```diff
@@ -54,12 +54,13 @@
 </div>
 
 <%= render BannerComponent.new(
-  title: t(".student-resources.topics_to_life.heading"),
+  title: t(".student-resources.topics_to_life.heading"), title_link: "/tech-careers-videos",
```
[the remark this answers is not in the document]
Comment (A-Wheeto) [reply app/views/pages/careers-support.html.erb]: Agreed

File: app/views/pages/careers-support.html.erb [hunk 2]
```diff
@@ -54,12 +54,13 @@
...
   text:  t(".student-resources.topics_to_life.content.html"),
   video: { url: cs_topic_video_1_url },
   media_column_size: :half,
   media_side: :left,
-  padding: 0
+  padding: 0,
+  button: { text: "Explore careers videos", link: "/tech-careers-videos", class: "ncce-button--white"},
```
[the remark this answers is not in the document]
Comment (A-Wheeto) [reply app/views/pages/careers-support.html.erb]: Agreed

File: app/views/pages/isaac_computer_science.html.erb [hunk 3]
```diff
@@ -85,7 +85,8 @@
   video: { url: cs_topic_video_1_url },
   media_column_size: :half,
   box_color: "white",
-  media_side: :right
+  media_side: :right,
+  button: { text: "Explore careers videos", link: "/tech-careers-videos", class: "ncce-button--white"},
```
[the remark this answers is not in the document]
Comment (A-Wheeto) [reply app/views/pages/isaac_computer_science.html.erb]: Agreed

File: spec/components/banner_component_spec.rb [hunk 4]
```diff
@@ -38,6 +43,10 @@
     it "should default to two thirds for content" do
       expect(page).to have_css(".banner-component__content-section--two-thirds")
     end
+
+    it "should have button" do
```
[the remark this answers is not in the document]
Comment (A-Wheeto) [reply spec/components/banner_component_spec.rb]: Agreed and updated

**Review by msquance-stem — APPROVED** [review]
LGTM

**Pull request merged**